### PR TITLE
Add client container for ppc64le

### PR DESCRIPTION
**Pull request by bedroge**

### CODE REVIEW

File: containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM docker.io/ppc64le/centos:7
+
+RUN yum install -y sudo vim openssh-clients cmake wget make unzip patch valgrind bzip2
+
+RUN yum install -y gcc gcc-c++ sqlite-devel python-devel libcap-devel libuuid-devel attr \
+ && yum install -y fuse-devel fuse3-devel zlib-devel openssl-devel
+
+# build CernVM-FS from source (no aarch64 Debian packages available)
+RUN  wget https://github.com/cvmfs/cvmfs/archive/cvmfs-2.7.5.tar.gz && \
```
Comment (terjekv) [inline containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le:9]: Should we have a variable for the CVMFS-version? 2.7.5 is repeated... a lot. Something like

```
ARG cvmfsversion=2.7.5 
```

And then interpolate it in the rest of the file?

File: containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  cd cvmfs*2.7.5/ && \
+  mkdir build && \
+  cd build && \
+  cmake .. -DBUILD_SERVER=no -DBUILD_SERVER_DEBUG=no -DDBUILD_SHRINKWRAP=no && \
+  make -j $(nproc) && \
+  sudo make install && \
+  cd / && \
+  rm -r cvmfs*2.7.5*
+
+RUN yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.3/cvmfs-config-eessi-0.2.3-1.noarch.rpm
```
Comment (terjekv) [inline containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le:20]: Looking at variables, should we have a variable for the config version and possibly the config package version (as they don't always match due to `-1`)?

File: containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le (hunk 1 above)
Comment (bedroge) [reply containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le:9]: I copied most of it from this one:
https://github.com/EESSI/software-layer/blob/master/Dockerfile.fuse-overlay-debian10-aarch64
But I agree that it's much cleaner to use a variable for this. I'll take a look at that (and then add the same improvements to the one in software-layer, and update the version to 2.8.0).

Seems like there's also still some weird issue with the container (I'm getting weird errors for one repo) that needs to be solved.

File: containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le (hunk 2 above)
Comment (bedroge) [reply containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le:20]: That makes sense too. We also were thinking about having some link that always points to the latest version, which is partly already made possible by Github, except for the version in the filename...

**Comment by bedroge** [comment]
Tested on a POWER9 node:
```
# specify Singularity configuration
export SINGULARITY_HOME="$EESSI_TMPDIR/home:/home/$USER"
export SINGULARITY_BIND="$EESSI_TMPDIR/var-run-cvmfs:/var/run/cvmfs,$EESSI_TMPDIR/var-lib-cvmfs:/var/lib/cvmfs"

# values to pass to --fusemount in singularity command
export EESSI_CONFIG="container:cvmfs2 cvmfs-config.eessi-hpc.org /cvmfs/cvmfs-config.eessi-hpc.org"
export EESSI_PILOT="container:cvmfs2 pilot.eessi-hpc.org /cvmfs/pilot.eessi-hpc.org"

singularity exec --fusemount "$EESSI_CONFIG" --fusemount "$EESSI_PILOT" docker://eessi/client-pilot:centos7-ppc64le ls /cvmfs/pilot.eessi-hpc.org
```

```
...
INFO:    Creating SIF file...
CernVM-FS: pre-mounted on file descriptor 3
CernVM-FS: pre-mounted on file descriptor 3
CernVM-FS: loading Fuse module... done
CernVM-FS: loading Fuse module... done
2020.10
2020.12
latest
```

@terjekv good to go?

**Review by boegel — APPROVED** [review]

**Pull request merged**